pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -1439,6 +1445,11 @@ pub mod pallet {
             .into())
         }
 
+        // TODO: this call should be removed prior to mainnet launch.
+        // It's super useful for testing purposes, but even though force is used in this pallet & works well,
+        // it won't apply to the inflation recalculation logic - which is wrong.
+        // Probably for this call to make sense, an outside logic should handle both inflation & dApp staking state changes.
+
```

Comment:
By testing do you perhaps mean local node testing for UI or unit tests?

Reply:
Only the local node testing - for myself and UI 🙂 

For unit test it's not needed. There are functions which simply _run_ blocks up to some value.